graphql_compiler/tests/test_compiler.py

```diff
@@ -3336,6 +3337,76 @@ def test_has_edge_degree_op_filter_with_fold(self):
         check_test_data(self, test_data, expected_match, expected_gremlin, expected_sql,
                         expected_cypher)
 
+    def test_is_null_op_filter(self):
+        test_data = test_input_data.is_null_op_filter()
+
+        expected_match = '''
+            SELECT
+                Animal___1.name AS `name`
+            FROM  (
+                MATCH  {{ class: Animal,
```

Comment:
nit: mind putting this on its own line, and using single spaces between `FROM` and `(`, and `MATCH` and `{{`? That way it'll match the structure of the other MATCH test cases.